nzedb/Binaries.php

```diff
@@ -724,27 +748,33 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 					// Get the current unixtime from PHP.
 					$now = time();
 
+					$xref = ($multiGroup === true ? sprintf('xref = CONCAT(xref, "\\n"%s ),', $this->_pdo->escapeString(substr($header['Xref'], 2, 255))) : '');
```

Comment:
Why would you add another column or table for something that is deleted after release has been created? This is used only for MGR, as those are posted to 8 or 9 groups with only one group in xref. This is done solely for NZB creation, so we have proper groups listed there.